src/layouts/error.vue

```diff
@@ -1,305 +1,63 @@
+        <ButtonV2
+          :text="$t('error_page_not_found_return_home')"
+          preset="primary"
+          :to="{ name: 'index' }"
         />
-      </template>
-      <template #body>
-        <Transition name="error-dialog-content-" appear="appear">
-          <main class="page-content error-dialog-content">
-            <i18n
-              v-if="isLoginErrorFromCivicLikerRegistration"
-              class="mt-16 text-24 font-600"
-              :path="`ERROR.LOGIN_NEEDED_TO_SUPPORT_CREATOR${referrer ? '_WITH_NAME' : ''}`"
-              tag="p"
-            >
-              <span
-                v-if="referrer"
-                class="whitespace-no-wrap"
-                place="creator"
-              >{{ referrer.displayName }}</span>
-            </i18n>
-            <template v-else-if="formattedTitle">
-              <h1 class="px-12 mt-16 text-28">{{ formattedTitle }}</h1>
-              <p class="mt-32 text-16 leading-1_5">{{ formattedMessage }}</p>
-            </template>
-            <p v-else class="mt-16 text-24 font-600">{{ formattedMessage }}</p>
-            <div class="px-12 mt-32 phone:px-0">
-              <div v-if="isLoginError">
-                <a
-                  class="btn btn--outlined"
-                  :href="getOAuthRegisterAPI"
-                  @click="onClickLogEvent('Register', 'RegisterSignUp', 'RegisterSignUp(error page)', 1)"
-                >{{ $t('signUp') }}</a>
-              </div>
-              <div v-if="['CIVIC_LIKER_TRIAL_EVENT_JOINED', 'CIVIC_LIKER_ALREADY_PAID'].includes(error.message)">
-                <NuxtLink
-                  class="btn btn--outlined"
-                  :to="{ name: 'civic' }"
-                >{{ $t('learnMore') }}</NuxtLink>
-                <NuxtLink
-                  v-if="error.message === 'CIVIC_LIKER_TRIAL_EVENT_JOINED'"
-                  class="btn btn--outlined"
-                  :to="{ name: 'civic-register' }"
-                >{{ $t('upgrade') }}</NuxtLink>
-              </div>
-              <!-- - Common action button-->
-              <template v-else>
-                <a
-                  v-if="isLoginError"
-                  class="mx-0 btn btn--plain btn--auto-size text-14"
-                  :href="getOAuthLoginAPI"
-                  @click="onClickLogEvent('Register', 'RegisterSignIn', 'RegisterSignIn(error page)', 1)"
-                >{{ $t('signIn') }}</a>
-                <button
-                  v-if="!error.isBackButtonHidden"
-                  class="mx-0 btn btn--plain btn--auto-size text-14"
-                  @click="onClickBackButton"
-                >{{ $t('back') }}</button>
-              </template>
-            </div>
-          </main>
-        </Transition>
-      </template>
-    </DialogLayout>
-  </Transition>
+        <ButtonV2
+          :text="$t('error_page_not_found_report_problem')"
+          preset="outline"
+          href="https://discord.com/channels/763001015712350231/814761730349596712"
```

Comment:
I think we can actually trigger crisp
https://github.com/likecoin/liker-land/pull/726